src/data_capture/video_capture/launch/capture.launch

```diff
@@ -2,18 +2,18 @@
 
     <arg name="image_topic" default="camera/color/image_raw"/>
     <arg name="is_record_topic" default="video_capture/is_record"/>
-    <arg name="is_memory_usage_exceeded_topic" default="data_capture/is_memory_usage_exceeded"/>
     <arg name="output_directory" default="/root/videos"/>
 
     <arg name="frames_per_second" default="30.0"/>
     <arg name="video_type" default="mp4"/>
     <arg name="video_dimensions" default="480p"/>
 
+    <param name="data_capture/is_memory_usage_exceeded_param" type="str" value="data_capture/is_memory_usage_exceeded">
```

Comment:
Could you make the parameter name more descriptive? For example, `data_capture/is_high_memory_usage_topic` or something similar?

Reply:
Done